final_prep/README.md

```diff
@@ -23,52 +23,52 @@ Some things to consider as you refactor include...
+- [x] Study/Project work time (usually about 25 hours/week outside of class time)
+- [x] Personal, Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- ![Image 3-4-22 at 12 23 PM](https://user-images.githubusercontent.com/99003546/156837161-18d46d7b-7f07-4878-b6af-f56bca46416d.jpg)
+- ![C5147885-5438-424D-9540-9CC3183D52ED](https://user-images.githubusercontent.com/99003546/156837267-915957c4-b7ca-409c-89af-7b881bc47192.jpeg)
+- ![B449EF8F-65F1-4976-A1B9-620016101A3E](https://user-images.githubusercontent.com/99003546/156837413-cc7c1531-1334-4936-9647-aa7ec9af3eaa.jpeg)
 
 ## Mentorship Prep
 
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 
-- [ ] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+- [x] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
+- [x] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+  - [x] [Add link to your gist here:](https://gist.github.com/sdecesari/1e7a1318e7fff2089d4ce1bf264c6e38)
 
 ## Lesson Prep
 
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
 
-- [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
-  - [ ] Add a link to your gist here:
+- [x] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
+  - [x] [Add a link to your gist here:](https://gist.github.com/sdecesari/fe6d4fc639fc8418b985959cd215124e)
 
 ## Group Work Prep
 
 As part of Turing's project-based learning approach, you will often be working in pairs or larger groups. In order to set yourself (and your team) up for success, it is important to ensure you are prepared to be an equitable contributor and teammate.
 
-- [ ] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
-  - [ ] Add a link to your gist here: 
+- [x] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
+  - [x] [Add a link to your gist here:](https://gist.github.com/sdecesari/e92a196537dd2b43ad67b1a70542581e)
```

Comment:
Based on this and Mentor DTR, you seem really thoughtful and like you're in a strong place to enter these working relationships! I'm glad to see you talking about having ADHD and your self-awareness around it!